bin/webpack.dev.js

```diff
@@ -66,6 +66,13 @@ module.exports = {
                 require.resolve('babel-preset-react'),
                 require.resolve('babel-preset-stage-0'),
               ],
+              plugins: [
+                [
+                  require.resolve('babel-plugin-import'), {
```

Comment:
must use the whole name, or it will not be an error 'Module build failed: ReferenceError: Unknown plugin "import" specified in "base" at 0'.